client/src/components/PropertiesList.tsx

```diff
@@ -60,16 +77,19 @@ const PropertiesList = (props) => {
                 },
                 {
                   Header: 'Zipcode',
-                  accessor: 'zip',
+                  accessor: d => d.zip,
+                  id: 'zip',
```

Comment:
react-table doesn't have great third-party typings--the `accessor` can be just a "string" or a function that takes an `Addr`, and as there's no type-checking on the string, I opted to make things a bit more verbose by providing a function instead.  This triggered an error from react-table claiming I had to also provide a string `id` if `accessor` was a function, so I did that, even though I don't really understand what the `id` is for.

Reply:
Hmmm, yeah not sure what the id field is either. I'd assume it's used in the html structure somewhere but I don't see it 🤷‍♂